python-avd/pyavd/_eos_designs/fabric_documentation_facts/__init__.py

```diff
@@ -112,7 +112,7 @@ def _topology(self) -> Topology:
                 else:
                     peer_ip_address = None
 
-                routed = get(ethernet_interface, "type") == "routed"
+                routed = get(ethernet_interface, "switchport.enabled") is not None and not get(ethernet_interface, "switchport.enabled")
```

Comment:
Why not just check for `get(ethernet_interface, "switchport.enabled") is False`?

Reply:
Fixed